src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -180,6 +180,13 @@
         <xsd:attribute name="cache" type="xsd:string" />
         <xsd:attribute name="enable-annotations" type="xsd:boolean" />
         <xsd:attribute name="static-method" type="xsd:boolean" />
+        <xsd:attribute name="mapping" type="validation_mapping" />
+    </xsd:complexType>
+
+    <xsd:complexType name="validation_mapping">
+        <xsd:sequence>
+            <xsd:element name="path" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
What about using `1` as the value for `minOccurs` (if you do not want to specify any paths you should omit the `mapping` element entirely imo)?